Dockerfile

```diff
@@ -1,11 +1,15 @@
 ARG PHP_VERSION=7.2
 FROM php:${PHP_VERSION}-cli
 
+# Install pickle to help manage extensions
```

Comment:
Can you revert this change please, we're going to stick with pecl for now

Reply:
Should I revert to what is currently in master?
![image](https://user-images.githubusercontent.com/10543911/101695294-93a7fa80-3a31-11eb-8248-2990f9bee2d3.png)

Reply:
Yes please